course_content/notebooks/numpy_intro.ipynb

```diff
@@ -921,30 +922,45 @@
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "Used without any further arguments, statistical functions simply reduce the whole array to a single value.  However, in practice we very often want to calculate over only *some* of the dimensions.  \n",
+      "Used without any further arguments, statistical functions simply reduce the whole array to a single value.  In practice, however, we very often want to calculate statistics over only *some* of the dimensions.\n",
+      "\n",
       "For example, given the above `daily_records` data, we might want :\n",
-      "  * (a) an average over all days, for each station *-- or --*\n",
-      "  * (b) an average over all stations, for each day."
+      "\n",
+      "* (a) an average over all days, for each station, or\n",
+      "* (b) an average over all stations, for each day."
      ]
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
       "#### Exercise nn.mm\n",
-      " * produce the two kinds of single-axis average mentioned above -- that is, (a) over all days at each station, and (b) over all stations for each day.  \n",
-      "   Hint: look at the documentation -- the term 'axis' in numpy refers to the number of a dimension.\n",
-      " * Create a sample 3-D array, to represent air temperature at given times, X and Y positions :\n",
-      "   * (a) how can you form a mean over all X and Y at each timestep ?  \n",
-      "   * (b) what shape does such result have ?\n",
-      "   * (c) how can you calculate the `np.ptp` statistic of this data in a similar form, i.e. for all X and Y at each timestep ?  \n",
+      "\n",
+      "* produce the two kinds of single-axis average mentioned above -- that is, (a) over all days at each station, and (b) over all stations for each day. (Hint: look at the documentation -- the term 'axis' in NumPy refers to the number of a dimension.)\n",
+      "* Create a sample 3-D array, to represent air temperature at given times, X and Y positions :\n",
+      "   1. how can you form a mean over all X and Y at each timestep ?  \n",
+      "   1. what shape does such result have ?\n",
+      "   1. how can you calculate the `np.ptp` statistic of this data in a similar form, i.e. for all X and Y at each timestep ?  \n",
       "     N.B. `ptp` means a peak-to-peak range, i.e. \"max(data) - min(data)\"."
      ]
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
+      "#### Statistics : Summary of key points\n",
```

Comment:
I moved this up as it seemed better to summarise stats before starting the somewhat discrete topic of masked arrays - this seems to fit better here. Happy to have an offline argument about the placing of this cell though!